recipe/meta.yaml

```
@@ -49,3 +49,4 @@ about:
 extra:
   recipe-maintainers:
   - prashantkhoje
+  - bdice
```

@jakirkham requested to be added to the maintainer list as well. @jakirkham please confirm in a comment.
```suggestion
  - bdice
  - jakirkham
```